packages/excalidraw/tests/export.test.tsx

```diff
@@ -51,7 +51,7 @@ describe("export", () => {
       blob: pngBlob,
       metadata: serializeAsJSON(testElements, h.state, {}, "local"),
     });
-    API.drop(pngBlobEmbedded);
+    await API.drop(pngBlobEmbedded);
```

Comment:
these `API.drop` calls (or perhaps something else here) tends to complain about missing `act()` but I couldn't figure out what's causing it.

Reply:
I can't see this warning at all when running the tests.

Reply:
For what it's worth, I've been switching my CPU clock (three steps) and ran the full suite 10+ times at various speeds. This issue does not surface.

I've seen 3 "still loading" expectation fails though at the lowest CPU setting.

Reply:
> I've seen 3 "still loading" expectation fails though at the lowest CPU setting.

interesting. I assume the CPU was so slow that the `waitFor()` timed out.